shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -410,6 +410,14 @@ fun String.extractShortTypeFromExpanded(): String =
      */
     this.substringAfterLast("/").substringAfterLast("#")
 
+fun String.isExpanded(): Boolean =
```

Comment:
Not that sure about that.
Is there a more efficient way to check if a String is a Fully Qualified Name ?

Reply:
you don't need to bother with this. just expand the type. if it's already expanded, you'll get it back as is (well, I'm almost pretty sure of this, check it to be sure).

Reply:
I confirm